cli/build.gradle

```diff
@@ -123,9 +124,36 @@ downloadPortal {
 	onlyIfNewer true
 }
 
+evaluationDependsOn(":extensions:maven-profile")
+
 jar {
+	File serviceDir = new File("$buildDir/META-INF/services")
+
+	doFirst {
+		serviceDir.deleteDir()
+		serviceDir.mkdirs()
+
+		for(file in project.sourceSets.main.resources.srcDirs) {
```

Comment:
Hey @christopherbryanboyd  I don't want to it it this way.  Instead just embed the maven-profile.jar inside of the blade.jar and then just modify the Extensions class to have the ability to look for extensions packaged into the blade.jar itself (similar to how we look inside blade.jar to find archetypes).  That way the BaseCommand file can remain unchanged.